src/.vscode/tasks.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I don't think this file should be in the repository.

Reply:
Yes, will delete this.